rwengine/src/engine/GameData.cpp

```diff
@@ -446,6 +446,7 @@ void GameData::loadModel(ModelID model) {
             slotname = name;
             break;
         case ModelDataType::PedInfo:
+        {
```

Comment:
"How did this ever work?" - normally my clang complains about such things.
Also I'd suggest moving the opening brace on the previous line as we do for `if` and functions. But it's ultimately @tsjost and @danhedron call